pkg/local_object_storage/engine/container.go

```diff
@@ -20,6 +20,10 @@ func (e *StorageEngine) ContainerSize(cnr cid.ID) (uint64, error) {
 		err  error
 		size uint64
 	)
+	if e.metrics != nil {
```

Comment:
> closed? I've not found any way to block, really

should be dropped then or try to write code keeping in mind that it can be blocked after some future feature

Reply:
It was itching for me as well, but I've kept it for now.